### DISCUSSION TITLE
当前的组件解析没有考虑到全局组件的情况

### DISCUSSION THREAD

**Discussion by chuifengji** [ctripcorp/wean#7] [General]
如题，那么全局组件依赖是不是要下发到每个子页面呢，虽然有点蠢，但是对现有代码的改动应该会少不少。

**Comment by yisar** [comment]
应该不需要下发，现在 wepack 的组件是挂在 window 上的全局变量上的，类似于 module federation
所以现在无非就是 app.json 负责挂载，页面负责调用

这块我重构一下，因为全局变量还存在一些问题

> 虽然有点蠢，但是对现有代码的改动应该会少不少。

放心，不怕改动大，追求架构稳定性更重要
